src/main/java/org/folio/event/service/UserTenantService.java

```diff
@@ -134,7 +134,7 @@ public Future<Void> validateUserAcrossTenants(User entity, User userFromStorage,
   }
 
   public Future<Void> validateUserAcrossTenants(User entity, Map<String, String> okapiHeaders, Conn conn, Context vertxContext) {
-    Predicate<User> predicate = user -> true;
+    Predicate<User> predicate = user -> ObjectUtils.notEqual(UserType.SYSTEM.getTypeName(), entity.getType());
```

Comment:
we need some test for this change: unit or karate. If we could not write unit test in mod-users maybe lets create new test just with mocks in mod-users or write karate

Reply:
covered by this karate pr: https://github.com/folio-org/folio-integration-tests/pull/1195